libs/utils/utils.js

```diff
@@ -1000,7 +1000,10 @@ export function scrollToHashedElement(hash) {
 }
 
 function logPagePerf() {
-  if (getMetadata('pageperf') !== 'on') return;
```

Comment:
I still wonder what's the ratio of pages where this will be set vs not. If we have more pages where we're not using it, I'd advocate we return ASAP without any sort of additional check.